### PR TITLE
iTeam dev listing

### PR DESCRIPTION
**Pull request by gmoore**
Please proofread. I'm an awful typist.

Also this will probably conflict with John's PR. I copied the `<h2>` through the `</b></u></p>` of his PR into mine.

### CODE REVIEW

**Comment by subelsky** [comment]
I fixed the merge conflicts and directly merged. Thanks!

I guess if this gets any more elaborate we'll convert to Jekyll - then we can have a template and each of these can be a separate include file

File: index.html [hunk 1]
```diff
@@ -106,6 +106,30 @@
               <li>You live in or near Baltimore, or would move here for the right opportunity.</li>
             </ul>
 
+            <h2>Current Openings</h2>
+
+            <p>We are always seeking talented programmers but in addition we are specifically in need of the following positions:</p>
+
+            <p><b><u>Integrations Developer:</b></u></p>
+
+            <p>At STAQ, we have a fantastic platform to consume and report on data for our customers. You job will be to write code to go out and get more data. The data we target is of widely disparate availability. Sometimes the work is  as simple as hitting an API endpoint and parsing some JSON. Other times, things get far more complicated and you need to dig into your coding toolkit to extract the data points we’re looking for.</p>
```
Comment (kbladow) [inline index.html]: Your job

work is  as simple <- remove spaces

File: index.html [hunk 2]
```diff
@@ -106,6 +106,30 @@
...
+
+            <p>We have a teaching and mentoring culture at STAQ, so if you have taught yourself some things and have a motor to learn more and more then <u>write to us and introcuse yourself and let's talk</u>.
```
Comment (kbladow) [inline index.html]: should motor be motivation?

introcuse -> introduce

File: index.html [hunk 3]
```diff
@@ -106,6 +106,30 @@
...
+
+            <p>Sound interesting? Check out this list and see if this sounds like you. </p>
+            <p>You...</p>
+            <ul>
+              <li>are eager to contribute. Our platform allows us to push to production early and often so you’ll be making an impact right away.</li>
```
Comment (kbladow) [inline index.html]: often, so you’ll be

File: index.html [hunk 4]
```diff
@@ -106,6 +106,30 @@
...
+              <li>don’t get discouraged by things not working right the first time</li>
+              <li>enjoy teamwork and a collaborative workplace</li>
+              <li>are ready to learn lots of things</li>
+              <li>have a little experience with Ruby</li>
```
Comment (kbladow) [inline index.html]: You might want to be more clear about what this means. (Your "little" might not be the same as someone else's.)

File: index.html [hunk 5]
```diff
@@ -106,6 +106,30 @@
...
+              <li>can communicate clearly (verbally and in writing)</li>
+              <li>like writing code and are always to looking to improve your programming skills.</li>
+              <li>live in or near Baltimore, or would move here for the right opportunity.</li>
```
Comment (kbladow) [inline index.html]: The only thing I feel like might be missing might be the idea of liking to solve problems.

**Comment by gmoore** [comment]
Great stuff thank you @kbladow

**Pull request closed without merging**